aten/src/ATen/core/dispatch/Dispatcher.h

```diff
@@ -508,8 +508,9 @@ inline Return Dispatcher::callWithDispatchKeySlowPath(const TypedOperatorHandle<
 template<class Return, class... Args>
```

Comment:
I will be surprised if making argument lists in these functions wider is worth whatever savings we're getting, particularly when we're statically linked and thus don't need tls_get_addr